pkg/agent/sysadvisor/plugin/inference/modelresultfetcher/borwein/borwein.go

```diff
@@ -354,7 +355,7 @@ func NewBorweinModelResultFetcher(fetcherName string, conf *config.Configuration
 ) (modelresultfetcher.ModelResultFetcher, error) {
 	if conf == nil || conf.BorweinConfiguration == nil {
 		return nil, fmt.Errorf("nil conf")
-	} else if !conf.PolicyRama.EnableBorweinModelResultFetcher {
+	} else if !conf.GetDynamicConfiguration().PolicyRama.EnableBorweinModelResultFetcher {
```

Comment:
Don't use GetDynamicConfiguration in init function which is initialized only once.